packages/api-gateway/src/drivers/http/routes/places/schema.js

```diff
@@ -20,6 +20,19 @@ const postSchema = {
   },
 };
 
+const deletePlaceSchema = {
+  description: 'This is the route to use for deleting a place',
+  tags: ['Places'],
+  body: {
+    type: 'object',
+    properties: {
+      id: { type: 'number' },
+    },
+    required: ['id'],
+  },
```

Comment:
Can you add the response spec here? we need to define the responses of our endpoints so people reading the documentation can know what to expect from the endpoint before calling it.
```suggestion
  response: {
    200: {
      type: 'object',
      properties: {
        message: { type: 'string' },
      },
    },
  },
```